base_station/gui/src/components/AutonTask.vue

```diff
@@ -179,6 +180,32 @@ export default {
   },
 
   created: function () {
+    setInterval(() => {
+      if(this.nav_status.nav_state_name == "Off"){
+        this.nav_state_color = "#4695FF"
```

Comment:
can this number #4695FF be changed to be a variable name for readability? is this supposed to be blue?